core/config/config_test.go

```diff
@@ -0,0 +1,38 @@
+package config
+
+import (
+	"chain/database/pg/pgtest"
+	"chain/database/sinkdb"
+	"chain/database/sinkdb/sinkdbtest"
+	"context"
+	"crypto/rand"
+	"encoding/hex"
+	"testing"
```

Comment:
nit: we group our imports with standard lib packages first, then vendored packages, then chain packages. like

```
import (
	"context"
	"crypto/rand"
	"encoding/hex"
	"testing"

	"chain/database/pg/pgtest"
	"chain/database/sinkdb"
	"chain/database/sinkdb/sinkdbtest"
)
```